tests/integration/docusaurus/expectations/how_to_create_and_edit_an_expectationsuite_domain_knowledge.py

```diff
@@ -0,0 +1,119 @@
+import pathlib
+import great_expectations as gx
+import tempfile
+
+temp_dir = tempfile.TemporaryDirectory()
+full_path_to_project_directory = pathlib.Path(temp_dir.name).resolve()
+data_directory = pathlib.Path(
+    gx.__file__,
+    "..",
+    "..",
+    "tests",
+    "test_sets",
+    "taxi_yellow_tripdata_samples",
+).resolve(strict=True)
+
+# <snippet name="tests/integration/docusaurus/expectations/how_to_create_and_edit_an_expectationsuite_domain_knowledge.py get_data_context">
+import great_expectations as gx
+
+context = gx.data_context.FileDataContext.create(full_path_to_project_directory)
+# </snippet>
+
+
+context.sources.add_pandas(name="my_datasource",).add_csv_asset(
+    name="my_data_asset", filepath_or_buffer="./data/yellow_tripdata_sample_2019-01.csv"
+)
+
+# <snippet name="tests/integration/docusaurus/expectations/how_to_create_and_edit_an_expectationsuite_domain_knowledge.py get_data_asset_and_build_batch_request">
+data_asset = context.get_datasource("my_datasource").get_asset("my_data_asset")
+batch_request = data_asset.build_batch_request()
+# </snippet>
```

Comment:
Since we aren't using a batch, can we get rid of this section?

Reply:
yup, you're absolutely right